scripts/coverage_report_sprout.py

```diff
@@ -20,15 +23,18 @@
     # TODO: Upstream support
     group = 'downstream-' + ''.join(args.version.split('.')[:2]) + 'z'
     sprout = SproutClient.from_config()
-    print('requesting an appliance from sprout for {}/{}'.format(group, args.version))
-    pool_id = sprout.request_appliances(group, version=args.version)
-    print('Requested pool {}'.format(pool_id))
+    logger.info('requesting an appliance from sprout for {}/{}'.format(group, args.version))
```

Comment:
**required** with just seeing the code i am under the impression this misses a stdout logger setup, please verify if thats the case

Reply:
It's at the top.

Reply:
https://github.com/ManageIQ/integration_tests/pull/6705/files#diff-722609d2929b16688397d81420cf8fc3R22

Reply:
**REQUIRED**
```python
logger.info('requesting an appliance from sprout for %s/%s', (group, args.version))

Reply:
@jamesooden this is a different file, hence my question

however i noted that this imports the other one so its actually fine it still could use a comment since logging setup is a magic action at a distance by the import

Reply:
Oh yes, I see your point Ronny, I'll add a comment (and I missed it was a different file).   I actually was setting it up in here at first but then Mike noted that it would be setup by including the library.   I actually don't like that there are two scripts report_coverage_jenkins.py and report_coverage_sprout.py, but I don't want to change that just yet.   However it seems like just one script is all that is necessary.

Reply:
a note is fine, its just something i'd like to see tracked and hopefully followed up on :+1: